libavpipe/src/avpipe_xc.c

```diff
@@ -4945,14 +4953,39 @@ avpipe_fini(
         }
     }
 
+    if ((*xctx)->params->copy_mpegts) {
+        cp_ctx_t *cp_ctx = &(*xctx)->cp_ctx;
+        coderctx_t *mpegts_encoder_ctx = &cp_ctx->encoder_ctx;
+        elv_warn("Freeing mpegts encoder context, closing io");
+        avio_close(mpegts_encoder_ctx->format_context->pb);
```

Comment:
Can we catch the return code from avio_close() - we should just log it as a warn.  It returns AVERROR so we can log with av_err2str()